### ISSUE TITLE
Remediation of RHEL-07-010370 breaks authentication with Windows

### ISSUE THREAD

**Issue by bkogami** [ComplianceAsCode/content#2706]
#### Description of problem:
In RHEL-07-010370; Set Deny for Failed Password attempts, I set the following according fix description.
- add the following line immediately after the pam_unix.so statement in the AUTH section:
auth [default=die] pam_faillock.so authfail deny=3 unlock_time=3600 fail_interval=900

When this is set and I ssh to the localhost (ssh user@localhost), I get a permission denied message.

I've been able to troubleshoot it down to [default=die]. Is there a different setting that could be used to get around this issue?
#### SCAP Security Guide Version:
0.1.33-5.el7

#### Operating System Version:
RHEL 7.4

#### Steps to Reproduce:
1. Used Redhat's Windows Integration guide to authenticate users with Windows AD.
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/windows_integration_guide/
2. Verified 'ssh user@localhost', entered password and was able to login through ssh.
3. Made a remote copy of /etc/pam.d/system-auth* and /etc/pam.d/password-auth* files.
4. Ran SCAP-Workbench and used remediation to fix security controls.
5. Perform ssh user@localhost. This time, I get a permission denied message
6. Compared the original password-auth and the new password-auth
7. Found that by changing [default=die] to sufficient, I could ssh into the localhost

#### Actual Results:
permission denied

#### Expected Results:
Login to localhost via ssh

**Comment by jamescassell** [comment]
see #2174

**Comment by redhatrises** [comment]
@bkogami can you provide the original and modified copies? Also, was the user account locked? In addition, what happens when you try `user@domain`?

**Comment by bkogami** [comment]
Hi Redhatrises,

I uploaded 2 files. password-auth-ac.orig and password-auth-ac.hardened for your review.

I also tried various ways of logging in through ssh. None worked until I changed [default=die] to either significant or requisite. When I did that, I was able to login from the console and use ssh.

I've tried "ssh bkogami@localhost.domain", "ssh bkogami\@domain@localhost", "ssh domain\bkogami@localhost". Nothing worked.

Bruce

[sample_pam.zip](https://github.com/OpenSCAP/scap-security-guide/files/1894594/sample_pam.zip)
